README.md

```diff
@@ -52,11 +53,14 @@ Moreover, if you migrate an _Enterprise Edition_, you'll be asked to provide a *
 To launch the tool, run:
 
 ```bash
-  $ php MigrationTool.php akeneo-pim:migrate
+  $ php Transporteo.php akeneo-pim:migrate
 ```
 
 then, let you guide ;) 
 
+At the end of the tool's job, you will have your database setuped with your data but we do not migrate your custom code.
+You will have to migrate it following this [Upgrade file](./UPGRADE-2.0.md)
```

Comment:
Add something like, we plan to automate this part in future release, stay tuned!